### PR TITLE
Update subscription_services_privacy_notice.md

### PR DESCRIPTION
**Pull request by DreaAlexandre**
Change to Monitor section of this Privacy Notice for transparency around the use of Google Analytics. Localization is needed for this Privacy Notice inline with supported languages for the previous version.

### CODE REVIEW

File: en/subscription_services_privacy_notice.md [hunk 1]
```diff
@@ -43,7 +43,9 @@ __Interaction data.__ Mozilla receives data about your interactions with the ser
...
-* [Formstack.](https://www.formstack.com/) Mozilla Monitor uses Formstack to capture feedback optionally provided by users on account cancellation. For information on Formstack’s privacy practices, see the [Formstack Privacy Policy](https://www.formstack.com/legal).   
+* [Formstack.](https://www.formstack.com/) Mozilla Monitor uses Formstack to capture feedback optionally provided by users on account cancellation. For information on Formstack’s privacy practices, see the [Formstack Privacy Policy](https://www.formstack.com/legal).
+  
+* [Google Analytics.](https://marketingplatform.google.com/about/analytics/) Mozilla Monitor uses Google Analytics to obtain metrics on how users engage with our websites. This helps us to improve site content. For more information about how Google uses your personal information, please visit [Google Analytics’ Privacy & Security](https://support.google.com/analytics/topic/2919631?hl=en&ref_topic=1008008&sjid=14989286036636170427-NA). You can install the [Google Analytics Opt-out Browser Add-on](https://tools.google.com/dlpage/gaoptout) to prevent data collection about your visits to the Service and prohibit data transmission to Google Analytics.
```
Comment (peiying2) [inline en/subscription_services_privacy_notice.md:48]: @DreaAlexandre The link to P & S article should drop `hl=en` in the URL so the site can be redirected to the language of the user's choice.

**Review by peiying2 — CHANGES_REQUESTED** [review]
The URL should drop `hl=en` so user gets to see the P&S article in the language of their choice, not English only as it currently stands.

**Comment by DreaAlexandre** [comment]
Good catch. Thanks. I only see it in one of the links and removed. I double
checked that the link still fully works without the reference on my end,
but a double check from you would be great too.

**Comment by peiying2** [comment]
@DreaAlexandre I did check before making the change request, :). Also, do we need to change the date of the document? I'm never sure when the date change is a warranted , minor or major update?

**Comment by DreaAlexandre** [comment]
Sorry, to be clear, I mean double check now that I removed the language
ref. I think you can update the date for this, yes.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**